### PR TITLE
feat: added quiz category page

### PR DESCRIPTION
**Pull request by sach10-create**

### CODE REVIEW

**Review by purohitdheeraj — APPROVED** [review]
Code looks good to merge

Preview link is not working

File: js/root.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+const themeIcon = document.querySelector(".theme-icon");
+const novaLogo = document.querySelectorAll(".logo");
+
+console.log(themeIcon);
```
Comment (santhoshpemmaka) [inline js/root.js:4]: If possible remove console.log statement. It's a bad practice we didn't write a console statement in the production.

**Review by sonishreyas — APPROVED** [review]
Lookd good

**Review by santhoshpemmaka — COMMENTED** [review]
Good to merge

**Pull request merged**